src/components/todo_filter/TodosFilter.tsx

```diff
@@ -0,0 +1,65 @@
+import cn from 'classnames';
+import { useContext, useState } from 'react';
+import { TodosContext } from '../../providers/TodosContext';
+import { Status } from '../../utils/status';
+
+export const TodosFilter = () => {
+  const { setFilter, todos, deleteAllTodos } = useContext(TodosContext);
+  const [selectFiler, setSelectFilter] = useState(Status.All);
+  const countActiveTodos = todos.filter(todo => !todo.completed).length;
+  const countCompletedTodos = todos.filter(todo => todo.completed).length;
```

Comment:
You can filter them once and get completed todos as all todos amount - active todos amount